src/web/index.html.ejs

```diff
@@ -3,17 +3,17 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width" />
-    <meta name="description" content="Theming demo for Firefox Quantum and beyond." />
+    <meta name="description" content="<%= htmlWebpackPlugin.options.description %>" />
     <!-- Open Graph meta tags-->
-    <meta property="og:title" content="Firefox Themer" />
-    <meta property="og:url" content="https://mozilla.github.io/Themer/" />
+    <meta property="og:title" content="<%= htmlWebpackPlugin.options.title %>" />
+    <meta property="og:url" content="<%= htmlWebpackPlugin.options.homepage %>" />
     <meta property="og:type" content="website" />
     <!-- Twitter card meta tags-->
-    <meta name="twitter:card" content="themer" />
+    <meta name="twitter:card" content="summary" />
     <meta name="twitter:site" content="@FxTestPilot" />
-    <meta name="twitter:title" content="Grab your parachute and get access to experimental Firefox features." />
-    <meta name="twitter:description" content="Theming demo for Firefox Quantum and beyond." />
-    <title>Firefox Themer</title>
+    <meta name="twitter:title" content="<%= htmlWebpackPlugin.options.title %>" />
```

Comment:
I'll have to re-read the docs to try and understand the difference betwixt a `twitter:title` and `twitter:description`, but tweak will change the `twitter:title` to "Firefox Themer" (versus the current parachute reference, which may have been a TestPilot relic).